### PR TITLE
Portfolio mobile version PR

### PR DESCRIPTION
**Pull request by andym80**
# Portfolio Mobile Version

### CODE REVIEW

**Comment by andym80** [comment]
This project was stopped by a Linters conflict **AnalyzeError EISDIR**. That Web hint linter didn´t give access to which errors must be solved. **[](https://loom.com/share/291a4218666c49698125ead269744853)**
This error was found in _./IMAGES _by a file named with a # sign and was changed by the letters Hashtag. 
Then all linters errors could be solved and refixed all style.css files into another repository.   
I _want to give thanks to all people involved to know about this problem._  Get simple instructions on the video link about this project.

**Comment by andym80** [comment]
[]((https://loom.com/share/291a4218666c49698125ead269744853)[](url)

**Comment by andym80** [comment]
[**[](https://loom.com/share/291a4218666c49698125ead269744853)**](url)

File: index.html [hunk 1]
```diff
@@ -1,48 +1,249 @@
...
+          <button type="button" class="button-project">HTML/CSS</button>
+          <button type="button" class="button-project">Ruby on Rails</button>
+          <button type="button" class="button-project">Javascript</button>
+        </div>
+        <div class="project-btn">
+          <button>See this project</button>
+          <span>
+            <img src="./images/icon-arrow-right.svg" alt="icon arrow"
+          /></span>
         </div>
```
Comment (OLIPLICHE) [inline index.html]: - [ ] Please from the project template you choose, this first block is your first project. Therefore, make sure that the content of the project is aligned with the flexbox. ( suggestion: Create a `<div>` block)
- Please this needs to be applied to each one of the 6 projects.

File: index.html [hunk 2]
```diff
@@ -1,48 +1,249 @@
...
-    </div>
+      <div class="skills-box">
+        <h3 class="project-name">Project name goes here</h3>
+        <div class="skills-style">
+          <button type="button" class="button-project">HTML/CSS</button>
+          <button type="button" class="button-project">Ruby on Rails</button>
+          <button type="button" class="button-project">Javascript</button>
+        </div>
+        <div class="project-btn">
+          <button>See this project</button>
```
Comment (OLIPLICHE) [inline index.html]: - [ ] Please the button (e.g., "See this projec") follow the button styling: Active, Hover, and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1))
b

File: index.html [hunk 3]
```diff
@@ -1,48 +1,249 @@
...
+      <img src="./images/laptop-front.svg" alt="laptop-front" />
 
-    </div>
+      <div class="skills-box">
+        <h3 class="project-name">Project name goes here</h3>
+        <div class="skills-style">
+          <button type="button" class="button-project">HTML/CSS</button>
+          <button type="button" class="button-project">Ruby on Rails</button>
+          <button type="button" class="button-project">Javascript</button>
+        </div>
```
Comment (OLIPLICHE) [inline index.html]: - [ ] Please the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 4]
```diff
@@ -1,48 +1,249 @@
...
       </div>
-      <div class="start-collab-box">
-        <button type="button">Start collaboration</button>
+      <div class="my-cv-box">
+        <button type="button">
+          Get my CV
+          <span>
+            <img src="./images/icon-download.svg" alt="download" />
+          </span>
+        </button>
```
Comment (OLIPLICHE) [inline index.html]: - [ ] Please the button (e.g., "Get my CV") follow the button styling: Active, Hover, and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9))

b

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @andym80 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️

- [x] Please, I strongly request that your Pull Request should have a short summary describing introduced changes in more detail

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @andym80, 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ 

You have submitted a project, that is **not passing the linters check**.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by danielufeli — APPROVED** [review]
Hi @andym80, 

Great work on making the changes requested by a previous reviewer,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Design ✔️
- Good use of Github flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**